### PR TITLE
Add 2 UI test cases which is replication rule edit and delete.

### PR DESCRIPTION
**Pull request by danfengliu**
Add 2 UI test cases which is replication rule edit and delete.
Signed-off-by: danfengliu <danfengl@vmware.com>

### CODE REVIEW

File: tests/resources/Harbor-Pages/Replication.robot [hunk 1]
```diff
@@ -187,11 +159,39 @@ View Job Log
...
+    Retry Select Object    ${name}
+    Retry Element Click    ${action_bar_edit}
+
+Find Item And Click Delete Button
+    [Arguments]    ${name}
+    Filter Object    ${name}
+    Retry Select Object    ${name}
+    Retry Element Click    ${action_bar_delete}
+
+Click Rule Name And Repication Edit Button
```
Comment (wy65701436) [inline tests/resources/Harbor-Pages/Replication.robot]: Edit Replication Rule By Name

File: tests/resources/Harbor-Pages/Replication.robot [hunk 2]
```diff
@@ -187,11 +159,39 @@ View Job Log
...
+    [Arguments]    ${name}
+    Switch To Registries
+    Switch To Replication Manage
+    Find Item And Click Edit Button  ${name}
+
+Click Rule Name And Repication Delete Button
```
Comment (wy65701436) [inline tests/resources/Harbor-Pages/Replication.robot]: Delete Replication Rule By Name

File: tests/resources/Harbor-Pages/Replication.robot [hunk 3]
```diff
@@ -187,11 +159,39 @@ View Job Log
...
+    [Arguments]    ${name}
+    Switch To Registries
+    Switch To Replication Manage
+    Find Item And Click Delete Button  ${name}
+
+Delete A Rule
```
Comment (wy65701436) [inline tests/resources/Harbor-Pages/Replication.robot]: Ensure Delete Replication Rule By Name

File: tests/resources/Util.robot [hunk 4]
```diff
@@ -150,6 +173,12 @@ Text Input
     Wait Until Element Is Visible And Enabled  ${element_xpath}
     Input Text  ${element_xpath}  ${text}
 
+Clear Field Of Characters
```
Comment (wy65701436) [inline tests/resources/Util.robot]: can we leverage the buildin method? https://github.com/goharbor/harbor/blob/master/tests/robot-indices/Selenium2Library.index#L11

File: tests/resources/Harbor-Pages/Replication.robot (hunk 3 above)
Comment (danfengliu) [reply tests/resources/Harbor-Pages/Replication.robot]: Done

File: tests/resources/Util.robot (hunk 4 above)
Comment (danfengliu) [reply tests/resources/Util.robot]: It does not work as expected, so I left old code in use.

**Pull request merged**